packages/medusa-test-utils/src/module-test-runner.ts

```diff
@@ -61,6 +63,7 @@ export function moduleIntegrationTestRunner({
   const moduleOptions: InitModulesOptions = {
     injectedDependencies: {
       [ContainerRegistrationKeys.PG_CONNECTION]: connection,
+      eventBusService: new MockEventBusService(),
```

Comment:
Suggestion: i suggest that we add dependencies as a config of the test runner and spread it here